test/integration/default/inspec/controls/tests.rb

```diff
@@ -4,3 +4,15 @@
   skip_control 'cis-network-access-2.2.2'
   skip_control 'windows-account-100'
 end
+
+control "control_smb1_is_disabled" do
```

Comment:
should we be a part of [windows-baseline](https://github.com/dev-sec/windows-baseline)?